lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -99,6 +99,18 @@ aaa_server_group:
     default_value: "default"
 
 dnsclient:
+  domain:
+    config_get: "show running-config all"
+    config_get_token: '/^ip domain-list (\S+)$/'
+    config_set: '<state> ip domain-list <name>'
+    default_value: ''
+
+  domain_vrf:
+    config_get: "show running-config all"
+    config_get_token: ['/vrf context %s/', '/ip domain-list (\S+)/']
```

Comment:
Hmm. I tried that, but it said `RuntimeError: Given 1 args, but token ["/^vrf context <vrf>$/", "/^\\s+ip domain-list (\\S+)$/"] requires 0`